website/install.html

```diff
@@ -1019,7 +1032,7 @@ <h2>
             alt="GNU/Linux"
             title="GNU/Linux"
           />
-          <img class="os-logo" src="macos.svg" alt="macOS" title="macOS" />
+          <img class="os-logo macos-logo-color" src="macos.svg" alt="macOS" title="macOS" />
```

Comment:
Nit: I'd call this class just `macos-logo`. The fact that it changes the color is an implementation detail.